quests/index.ts

```diff
@@ -94,7 +94,7 @@ async function cloneWorld(worldName: string, templateWorldName: string) {
     }
     const world = utils.world(worldName)
     log(`World clone complete for ${worldName}`)
-    return new Promise(resolve => setTimeout(() => resolve(world), 1))
+    return new Promise(resolve => setTimeout(() => resolve(world), 100))
```

Comment:
This approach is not reliable. As you mentioned yest, if the server is under additional load, 100 ms may not be long enough. 

I think the correct approach is to find a reliable why to check if the world is "ready", and then proceed. In previous code I was using `utils.world(name)` to test if the world is "ready". 

`utils.world(name)` returns `null`/`undefined` it the world is not ready: 
https://github.com/walterhiggins/ScriptCraft/blob/development/src/main/js/modules/utils/utils.js#L91